osf/external/gravy_valet/compat.py

```diff
@@ -0,0 +1,87 @@
+import enum
+
+import dataclasses
+
+from addons.box.apps import BoxAddonAppConfig
+from osf.models import Node, OSFUser
+from . import request_helpers as gv_requests
+
+
+class _LegacyConfigsForImps(enum.Enum):
+    """Mapping from a GravyValet StorageImp name to the Addon config."""
+    box = BoxAddonAppConfig
```

Comment:
Plan per discussion the other day is to add `wb_key` or similar to the `ExternalStorageService` instead of using data in ways it wasn't meant to be used